ch.threema.threema-web-desktop.yml

```diff
@@ -0,0 +1,74 @@
+app-id: ch.threema.threema-web-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    DEV_ENV: development
+    threema_web_version: threema-web-2.3.19
+modules:
+  - name: threema-web-desktop
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/threema-web-desktop/flatpak-node/cache
+        npm_config_cache: /run/build/threema-web-desktop/flatpak-node/npm-cache
+        npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: 'true'
+    sources:
+      - type: git
+        url: https://github.com/threema-ch/threema-web-electron.git
+        branch: develop
```

Comment:
```suggestion
```

Never a branch

Reply:
Thanks for the quick review!

I went after the Flatpak Building guide which suggested adding a branch when using a commit to make sure the branch is "at this commit":

> The commit to use from the git repository. If branch is also specified, then it is verified that the branch/tag is at this specific commit. This is a readable way to document that you're using a particular tag, but verify that it does not change.

(https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html?highlight=branch#)

I don't have particular feelings on including it though :-)

Reply:
no. this is outdated. (can't fix it). Build will fail if any commit is added to the branch, because it's a branch,

Reply:
I see, will change this.

Reply:
Ah, makes sense.  There are differences in the requirements/goals of Flatpak the tool and Flathub as a repo.  Could perhaps be worded differently, maybe I get around opening a PR there some day to clarify this.